test/spec/FileTreeView-test.js

```diff
@@ -29,9 +29,9 @@ define(function (require, exports, module) {
 
     var FileTreeView      = require("project/FileTreeView"),
         FileTreeViewModel = require("project/FileTreeViewModel"),
-        Preact            = require("preact-compat"),
+        Preact            = require("thirdparty/preact"),
         Immutable         = require("thirdparty/immutable"),
-        PTU               = require("preact-test-utils"),
+        PTU               = require("thirdparty/preact-test-utils"),
```

Comment:
I would recommend having a more readable name instead of `PTU`, `PreactTestUtils` may be.